### ISSUE TITLE
SPO Page modernization switch to select default user if existing user account does not exist

### ISSUE THREAD

**Issue by TuomasPakkanen** [pnp/pnpframework#123]
Current modernization engine has the switch KeepPageCreationModificationInformation, but it does not work perfectly if the existing modifier/creator no longer works at the organization and his/her account has been deleted from the directory. In this case, the engine sets the modified/created time stamps to now and replaces the modified by/created by information by the user who is running the modernization. Hopefully this issue will be fixed in the new engine. Few ideas:
1) keep original modified/created timestamps regardless
2) Allow setting specific account if existing user does not exist
3) Allow setting only timestamps

**Comment by TuomasPakkanen** [comment]
Further info at:
https://github.com/pnp/pnpframework/issues/108

**Comment by jansenbe** [comment]
@TuomasPakkanen : alternatively you can also force the author/editor information via PowerShell: see https://github.com/pnp/powershell/discussions/158 for some thread on that (we did some changes to make this easier in PS).

**Comment by TuomasPakkanen** [comment]
> @TuomasPakkanen : alternatively you can also force the author/editor information via PowerShell: see [pnp/powershell#158](https://github.com/pnp/powershell/discussions/158) for some thread on that (we did some changes to make this easier in PS).

Yeah. The problem with this case I think is that the original modified and created time stamps do not get migrated over if the existing user does not exist anymore. Original user info itself not important in my one customer case, but the timestamps are.

**Comment by jansenbe** [comment]
yes, but this approach allows you to detect the non existing user and replace with a default user while keeping the original dates
